rpcs3/rpcs3qt/call_stack_list.cpp

```diff
@@ -0,0 +1,37 @@
+﻿#include "call_stack_list.h"
+
+constexpr auto qstr = QString::fromStdString;
+
+call_stack_list::call_stack_list(QWidget* parent) : QListWidget(parent)
+{
+	setEditTriggers(QAbstractItemView::NoEditTriggers);
+	setContextMenuPolicy(Qt::CustomContextMenu);
+	setSelectionMode(QAbstractItemView::ExtendedSelection);
+
+	// connects
+	connect(this, &QListWidget::itemDoubleClicked, this, &call_stack_list::OnCallStackListDoubleClicked);
+}
+
+void call_stack_list::UpdateCPUData(std::weak_ptr<cpu_thread> cpu, std::shared_ptr<CPUDisAsm> disasm)
+{
+	this->cpu = cpu;
+}
+
+void call_stack_list::HandleUpdate(std::vector<u32> call_stack)
+{
+	clear();
+
+	for (auto addr : call_stack)
+	{
+		QString callStackItemText = qstr(fmt::format("0x%08llx", addr));
```

Comment:
```suggestion
		const QString call_stack_item_text = qstr(fmt::format("0x%08llx", addr));
```

Reply:
style's the same in breakpoint_list.cpp btw (gonna change it anyway)